clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -3302,6 +3302,9 @@ void ItaniumCXXABI::EmitThreadLocalInitFuncs(
       CharUnits Align = CGM.getContext().getDeclAlign(VD);
       Val = Builder.CreateAlignedLoad(Var->getValueType(), Val, Align);
     }
+    if (Val->getType() != Wrapper->getReturnType()) {
+      Val = Builder.CreateAddrSpaceCast(Val, Wrapper->getReturnType());
+    }
```

Comment:
No braces around one-line `if`.

Reply:
Don't think you need the condition, CreateAddrSpaceCast should be a no-op if the types match